src/safeds/data/tabular/transformation/_one_hot_encoder.py

```diff
@@ -174,21 +201,29 @@ def inverse_transform(self, transformed_table: Table) -> Table:
             If the transformer has not been fitted yet.
         """
         # Transformer has not been fitted yet
-        if self._wrapped_transformer is None or self._column_names is None:
+        if self._column_names is None or self._value_to_column is None:
             raise TransformerNotFittedError
 
-        data = transformed_table._data.copy()
-        data.columns = transformed_table.column_names
+        # Make a copy of the table:
+        # TODO: change to copy method once implemented
+        new_table = transformed_table.remove_columns([])
```

Comment:
Is the copy really necessary? At first glance, none of the operations you call on `new_table` mutate its internal state.